node/src/main/resources/defaults.conf

```diff
@@ -123,6 +123,27 @@ api-server {
 
   # Enable reporting API
   enable-reporting = false
+
+  # Sets a custom keepalive time, the delay time for sending next keepalive ping
+  keepalive-time = 7200000
+
+  # Sets a custom keepalive timeout, the timeout for keepalive ping requests
+  keepalive-timeout = 20000
+
+  # The most aggressive keep-alive time clients are permitted to configure.
+  # The server would close the connection if clients exceeding this rate
+  permit-keepalive-time = 300000
+
+  # Sets a custom max connection idle time, connection being idle for longer than which will be gracefully terminated
+  max-connection-idle = 9223372036854775807
+
+  # Sets a custom max connection age, connection lasting longer than which will be gracefully terminated
+  max-connection-age = 9223372036854775807
+
+  # Sets a custom grace time for the graceful connection termination. Once the max connection age
+  # is reached, RPCs have the grace time to complete. RPCs that do not complete in time will be
+  # cancelled, allowing the connection to terminate
+  max-connection-age-grace = 9223372036854775807
```

Comment:
https://github.com/grpc/grpc-java/blob/1c50bcafd5c5faa671f3731df24ea12878029f0a/netty/src/main/java/io/grpc/netty/NettyServerBuilder.java#L67-L107
Just want to share the default value. I don't change the default values

Reply:
Can we use `FiniteDuration` type like with other time values? It's much less chance to make an error.
```scala
val duration = FiniteDuration(Long.MaxValue, TimeUnit.NANOSECONDS)
// duration: scala.concurrent.duration.FiniteDuration = 9223372036854775807 nanoseconds

val x1 = Long.MaxValue
val x2 = maxDuration.toNanos
val x3 = maxDuration.toMillis
// x1: Long = 9223372036854775807
// x2: Long = 9223372036854775807
// x3: Long = 9223372036854
```